go/cmd/cosmos/main.go

```diff
@@ -23,6 +23,7 @@ type Config struct {
 	GRPCURL string `mapstructure:"grpcUrl"`
 	LCDURL  string `mapstructure:"lcdUrl"`
 	RPCURL  string `mapstructure:"rpcUrl"`
+	Scheme  string `mapstructure:"scheme"`
```

Comment:
added Scheme as cosmos nodes are http by default, datahub is https

Reply:
thinking it might be best to update our config to include fully qualified uri string and we can `url.Parse()` where needed to pluck out scheme/port/etc

Reply:
sounds good and simpler